LocaleUtils/index.js

```diff
@@ -7,7 +7,10 @@ export async function assertNoEnglishLeaks(mockFile, viewText, noTranslationCont
   }
   // Exclude all elements with a class of no-translate. Example phone numbers
   if (noTranslationContent.length) {
-    viewText = viewText.split(noTranslationContent).join('');
```

Comment:
Can we also move `LocaleUtils/index.js` outside the root of the project and into `playground`?

Reply:
We could, but its also used by the EnglishLeaks.spec file which is within testcafe. I wanted a common place that made sense for both the playground and our testcafe specs

Reply:
We already share mocks from the playground in our testcafe specs - so I'd suggest we nest it there for the time being.

Once we migrate to a mono-repo, we'll have a better location for shared files like this. 😄